services/reloader/0.0.119/defaults/cm.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
There seems to be some difference on the value of `values.yaml` here

Reply:
it looks like the only difference is that the cm is no longer broken up into newlines